assignments/objects.js

```diff
@@ -17,28 +17,84 @@ const example = {
   "gender": "F"
 }
 
+
+
+const Mitzi = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+
+}
+
+
+const Keenan = {
+  "id": 2,
+  "name": "Keenan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+  "speak": "Hey there",
+}
+
+const Keven = {
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+}
+
+const Gannie = {
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+}
+
+const Antonietta = {
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "M",
+  multiplyNums(foo, bar) {
+  	return foo * bar;
+  }
+ }
+
 // Write your intern objects here:
 
 
+
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
 
+console.log(Mitzi.name);
+
 // Kennan's ID
 
+console.log(Keenan.id);
+
 // Keven's email
 
+console.log(Keven.email);
+
 // Gannie's name
 
+console.log(Gannie.name);
+
 // Antonietta's Gender
 
+console.log(Antonietta.gender);
+
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
+
+console.log(Keenan.speak);
 
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
-//console.log(antonietta.multiplyNums(3,4));
+
+console.log(Antonietta.multiplyNums(3,4));
```

Comment:
could also define the method here, such as `Antonietta.multiplyNumbs = (a,b) => {...`